applications/openshift/api-server/api_server_encryption_provider_cipher/rule.yml

```diff
@@ -37,21 +43,19 @@ ocil_clause: '<tt>aescbc</tt> is not configured as the encryption provider'
 
 ocil: |-
     Run the following command:
-    <pre>$ oc get apiserver cluster -ojson | jq -r '.spec.encryption.type'</pre>
+    <pre>$ oc get --raw {{.var_apiserver_encryption_path}} | jq {{.var_apiserver_encryption_filter}} </pre>
     The output should return <tt>aescbc</tt> as the encryption type.
 
 warnings:
 - general: |-
-    {{{ openshift_cluster_setting("/apis/config.openshift.io/v1/apiservers/cluster") | indent(4) }}}
+    {{{ openshift_filtered_cluster_setting({custom_api_path: dump_path}) | indent(4) }}}
 
 template:
   name: yamlfile_value
   vars:
     ocp_data: "true"
-    entity_check: "at least one"
-    filepath: '/apis/config.openshift.io/v1/apiservers/cluster'
-    yamlpath: '.spec.encryption.type'
-    values:
-    - value: 'aescbc'
-      type: "string"
-      operation: "pattern match"
+    entity_check: "all"
```

Comment:
why did we check `at least one` and now we check `all`? (It doesn't seem to be wrong because the new check seems tighter, just curious.)

Reply:
It should only return one item, so it won't make a difference here